cpplite/cpplite.project/src/org/netbeans/modules/cpplite/project/ui/wizard/CPPLiteProjectWizardIterator.java

```diff
@@ -78,6 +80,7 @@ public Set instantiate() throws IOException {
         Preferences prefs = CPPLiteProject.getRootPreferences(projectDirectory);
         prefs.putBoolean(CPPLiteProject.KEY_IS_PROJECT, true);
         settings.getBuildConfig().save(getBuildPreferences(projectDirectory));
+        getRootPreferences(projectDirectory).put(KEY_COMPILE_COMMANDS, settings.getEditorConfigPath());
```

Comment:
I suggest to use the following, to be inline with the existing code in line 81:

```suggestion
        prefs.put(KEY_COMPILE_COMMANDS, settings.getEditorConfigPath())
```